# Request (ไม่มี input)
---
# Response
bool success
string message